src/commands/server/delete.ts

```diff
@@ -58,6 +59,28 @@ export default class Delete extends Command {
           }
         }
       },
+      {
+        title: 'Delete the CR eclipse-che of type checlusters.org.eclipse.che',
+        task: async (_ctx: any, task: any) => {
+          if (await kh.crdExist('checlusters.org.eclipse.che') &&
+            await kh.cheClusterExist('eclipse-che', flags.chenamespace)) {
+            await kh.deleteCheCluster('eclipse-che', flags.chenamespace)
+            await cli.wait(2000) //wait a couple of secs for the finalizers to be executed
+            task.title = await `${task.title}...OK`
+          } else {
+            task.title = await `${task.title}...CR not found`
+          }
+        }
+      },
+      {
```

Comment:
I wonder whether it would be better to delete the CRD *after* the `che-operator` deployment has been scaled down and deleted.

Reply:
That's what I did at the beginning but the CR could not be deleted. The CR deletion needs to operator because the finalizer is executed by the che-operator right?

Reply:
No, I'm speaking of deleting the CRD (line 76 and the following), not CR (line 63)

Reply:
Oh I see. Yes of course, let me change that.

Reply:
Done

Reply:
Thanks !